libtransact/src/database/sqlite.rs

```diff
@@ -83,6 +83,21 @@ impl SqliteDatabase {
 
         Ok(Self { pool })
     }
+
+    pub fn vacuum(&self) -> Result<(), SqliteDatabaseError> {
```

Comment:
The documentation for this method should probably be a bit more detailed since its meaning is specific to Sqlite.

Reply:
updated.